Documentation/states.md

```diff
@@ -12,15 +12,14 @@ The desired and last known states are exposed in the `DSTATE` and `STATE` column
 
 The `fleetctl` commands to act on units change the *desired state* of a unit. fleet itself is then responsible for performing the necessary state transitions to move a unit to the desired state. The following table explains the relationship between each `fleetctl` command and unit states.
 
-| Command | Desired State | Valid Previous States |
-|---------|--------------|-----|
-| `fleetctl submit`  | `inactive`  | `(unknown)`
```

Comment:
I know it's probably not _technically_ correct, but I'd lean towards saying "none" vs "unknown", but I don't feel strongly about it.